base/expr.jl

```diff
@@ -61,6 +61,16 @@ Takes the expression `x` and returns an equivalent expression with all macros re
 """
 macroexpand(x::ANY) = ccall(:jl_macroexpand, Any, (Any,), x)
 
+"""
+@macroexpand
```

Comment:
missing indent, and would have to be added to rst if this gets merged before #18588